spec/index.html

```diff
@@ -671,7 +671,7 @@ <h3>Algorithm</h3>
 
       <ol>
         <li id="iia-1">If there is already an issued identifier for
-          <var>existing identifier</var> in <a>issued identifiers list</a> for <var>I</var>,
+          <var>existing identifier</var> in <a>issued identifiers list</a> for <var>existing identifier</var>,
```

Comment:
Hmm, should it be this instead?

```suggestion
          <var>existing identifier</var> in <a>issued identifiers list</a> of <var>I</var>,
```

It seems like the variable wasn't wrong, but the language was. I think what we're trying to check here is if there is an issued identifier for a `existing identifier` in `I's list of issued identifiers`. So we could use `of` or the possessive language I just used -- whichever seems better to you (unless I'm just confused of course!).